### PR TITLE
Color scheme generator(workshop bounty)

### PR DESCRIPTION
**Pull request by giridhar7632**
Follows all the guidelines and link for video: https://hackclub.slack.com/archives/C01A9V5LPD0/p1606886323046400

### CODE REVIEW

**Review by faisalsayed10 — APPROVED** [review]
Just 1 change and everything's perfect! woohoo, nice job!

File: workshops/color_scheme_generator/README.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
+---
+name: "Color Scheme Generator"
+description: "Create color schemes using React Hooks"
+author: "@giridhar7632"
+---
+
+# Color Scheme Generator
+
+In this workshop we are going to use [React Hooks](https://reactjs.org/docs/hooks-intro.html). We're building a colorful app - 'random-color scheme generator' using react. It looks something like this.
```
[the remark this answers is not in the document]
Comment (giridhar7632) [reply workshops/color_scheme_generator/README.md]: just did!

File: workshops/color_scheme_generator/README.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+## Prerequisites
+
+Before starting this workshop, you should have some knowledge about [React.js](https://reactjs.org) and some [ES6](https://www.w3schools.com/Js/js_es6.asp) features of JavaScript.
+
+React Fundamentals
+ - [JSX](https://reactjs.org/docs/introducing-jsx.html)
+ - [Components and Props](https://reactjs.org/docs/components-and-props.html)
+ - [State and LifeCycle](https://reactjs.org/docs/state-and-lifecycle.html)
+
+If you are familiar with these, Let’s get going.
```
Comment (sampoder) [inline workshops/color_scheme_generator/README.md:24]:
```suggestion
If you are familiar with these, let’s get going.
```

**Review by sampoder — CHANGES_REQUESTED** [review]
I trust in Fayd's technical review, so I haven't performed a full one. However I would very carefully check the grammar of the workshop.

Thanks! 🚀

**Comment by giridhar7632** [comment]
@sampoder I checked grammar 😁

File: workshops/color_scheme_generator/README.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+- Easier to read, test, and debug.
+- Have better performance.
+- More reusable.
+
+> **Note:** Hooks don’t work inside classes. But you can use them instead of writing classes.
+
+## Setup
+
+For setting up the project fork the repl [here](https://repl.it/@Giridharhackclu/color-scheme-generator-starter#README.md). The starter repl contains ['create-react-app'](https://github.com/facebook/create-react-app) installed. There are some basic styles applied, you will get cleared along with the workshop.
```
Comment (sampoder) [inline workshops/color_scheme_generator/README.md:40]: @giridhar7632 please review the phrasing very closely, here's an example of a grammar / phrasing mistake.

**Review by sampoder — CHANGES_REQUESTED** [review]
Closing to clean up the list, feel free to resubmit when you feel it's ready.

**Pull request closed without merging**